src/components/Switch/Switch.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+const Switch = ({ isOn, handleToggle }) => {
+  return (
+    <>
```

Comment:
I guess usually you would have a `<div className="">` here to get some nice styling going.